src/contents/all/components/knot/Knot/Knot.tsx

```diff
@@ -82,6 +82,17 @@ const Knot = props => {
         }
       </div>
       <div className="knotMeta">
+        <Moment
+          filter={(value) =>
+            value.replace(/^a few seconds ago/g, 'just now')
+            .replace(/^a /g, '1 ')
+            .replace(/^an /g, '1 ')
+            .replace("minute", 'min')
+          }
+          fromNow
+        >
+          {props.knot.updatedAt ? props.knot.updatedAt : props.knot.postedAt}
```

Comment:
I like this, but I think it would be good to keep the postedAt as the default then add updatedAt as smaller and stylized next to it. Sort of like what we do for edited responses in mainframe.

So it would be like "3 days ago `edited 1 day ago`"  where the edited is smaller and lighter in color:

<img width="351" alt="Screen Shot 2020-07-13 at 3 41 57 PM" src="https://user-images.githubusercontent.com/217719/87360651-73761d00-c51f-11ea-998f-b44c84165b5c.png">